### PR TITLE
swaps: i18n token sections

### PR DESCRIPTION
**Pull request by skylarbarrera**
Fixes TEAM2-525
Figma link (if any):

## What changed (plus any additional context for devs)

adding i18n for token section types

## Screen recordings / screenshots

https://cloud.skylarbarrera.com/Screen-Shot-2022-10-14-14-17-34.58.png

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels? (`team1/team2`, `critical path`, `release`, `dev QA`)
- [x] Did you test both iOS and Android?
- [x] If your changes are visual, did you check both the light and dark themes?
- [x] Added e2e tests? If not, please specify why
- [x] If you added new critical path files, did you update the CODEOWNERS file?
- [x] If no `dev QA` label, did you add the PR to the QA Queue?

### CODE REVIEW

File: src/languages/_french.json [hunk 1]
```diff
@@ -423,7 +423,16 @@
...
       "swapping_for_prefix": "Swapping for :)",
-      "view_details": "View Details :)"
+      "view_details": "View Details :)",
+      "token_sections": {
+        "bridgeTokenSection": "􀊝 Bridge",
+        "crosschainMatchSection": "􀤆 On other networks",
+        "favoriteTokenSection": "􀋃 Favorites",
+        "lowLiquidityTokenSection": "􀇿 Low Liquidity",
+        "unverifiedTokenSection": "􀇿 Unverified",
+        "verifiedTokenSection": "􀇻 Rainbow Verified",
```
Comment (estebanmino) [inline src/languages/_french.json:433]: new design just says verified

**Review by estebanmino — COMMENTED** [review]
lgtm but conflicts 😓

File: src/languages/_french.json (hunk 1 above)
Comment (skylarbarrera) [reply src/languages/_french.json]: fixed ty

File: src/helpers/tokenSectionTypes.ts [hunk 2]
```diff
@@ -1,9 +1,9 @@
 export default {
-  bridgeTokenSection: '􀊝 Bridge',
-  crosschainMatchSection: '􀤆 On other networks',
-  favoriteTokenSection: '􀋃 Favorites',
-  lowLiquidityTokenSection: '􀇿 Low Liquidity',
-  unverifiedTokenSection: '􀇿 Unverified',
-  verifiedTokenSection: '􀇻 Verified',
-  unswappableTokenSection: '􀘰 No trade routes',
+  bridgeTokenSection: 'bridgeTokenSection',
```
Comment (estrattonbailey) [inline src/helpers/tokenSectionTypes.ts:2]: Idk if this is doing a whole lot for us atm, but all good for now! It's got me thinking about how we could strongly type the `lang` util 🤔

**Review by estrattonbailey — APPROVED** [review]

**Review by salman-rb — APPROVED** [review]

**Pull request merged**